doc/shared_library_howto/howto.txt

```diff
@@ -0,0 +1,107 @@
+
+Compiling and Using Theano functions as shared libraries
+========================================================
+
+First one important note: this functionality is experimental, so not every
+features works, and some may not work as you expected.
+
+This functionality was only be tested on CPU code on a Linux machine.
+
+For now, shared variables not works.
+
+Clearly every node of function that you want to use as shared library must
+have a C implementation.
+ 
+To compile a function as shared library, you must use the CLinker with the the
```

Comment:
as shared library -> as a shared library

must use the CLinker -> must use CLinker